znai-reactjs/src/doc-elements/page/Support.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react'
+import {jsonPromiseWithUrl} from "../../utils/json"
+
+class Support extends Component {
+    constructor(props) {
+        super(props)
+        const support = this.props.docMeta.support
+        const link = (support && support.link) ? support.link : null
+        this.state = {link: link}
+    }
+
+    componentDidMount() {
```

Comment:
this may not be ideal as it most likely will fetch on every page change. Initialization can be moved to `docMeta.js` potentially.

Reply:
Fetching is done in docMeta, but the actual evaluation is done in Support due to the nature of async.